Dockerfile.analytics

```diff
@@ -15,5 +15,91 @@ RUN apk --no-cache --update-cache add gfortran build-base wget freetype-dev libp
 RUN ln -s /usr/include/locale.h /usr/include/xlocale.h
 
 RUN pip3 install matplotlib==2.0.2 \
-psycopg2==2.7.2 \
-seaborn==0.8
+    psycopg2==2.7.2 \
+    seaborn==0.8
+
+# Based on https://github.com/tatsushid/docker-alpine-py3-tensorflow-jupyter/blob/master/Dockerfile
+# Changes:
+# - Bumping versions of Bazel and Tensorflow
+# - Add -Xmx to the Java params when building Bazel
+# - Disable TF_GENERATE_BACKTRACE and TF_GENERATE_STACKTRACE
+
+ENV JAVA_HOME /usr/lib/jvm/java-1.8-openjdk
+ENV LOCAL_RESOURCES 2048,.5,1.0
+ENV BAZEL_VERSION 0.10.0
+RUN apk add --no-cache python3 python3-tkinter py3-numpy py3-numpy-f2py freetype libpng libjpeg-turbo imagemagick graphviz git
+RUN apk add --no-cache --virtual=.build-deps \
+        bash \
+        cmake \
+        curl \
+        freetype-dev \
+        g++ \
+        libjpeg-turbo-dev \
+        libpng-dev \
+        linux-headers \
+        make \
+        musl-dev \
+        openblas-dev \
+        openjdk8 \
+        patch \
+        perl \
+        python3-dev \
+        py-numpy-dev \
+        rsync \
+        sed \
+        swig \
+        zip \
+    && cd /tmp \
+    && pip3 install --no-cache-dir wheel \
+    && $(cd /usr/bin && ln -s python3 python)
+
+# Bazel download
+RUN curl -SLO https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-dist.zip \
+    && mkdir bazel-${BAZEL_VERSION} \
+    && unzip -qd bazel-${BAZEL_VERSION} bazel-${BAZEL_VERSION}-dist.zip
+
+# Bazel install
+RUN cd bazel-${BAZEL_VERSION} \
+    && sed -i -e 's/-classpath/-J-Xmx8192m -J-Xms128m -classpath/g' scripts/bootstrap/compile.sh \
+    && cat scripts/bootstrap/compile.sh \
+    && bash compile.sh \
+    && cp -p output/bazel /usr/bin/
+
+# Build Tensorflow
+ENV TENSORFLOW_VERSION 1.7.0
+RUN cd /tmp \
+    && curl -SL https://github.com/tensorflow/tensorflow/archive/v${TENSORFLOW_VERSION}.tar.gz \
+        | tar xzf - \
+    && cd tensorflow-${TENSORFLOW_VERSION} \
+    && : musl-libc does not have "secure_getenv" function \
+    && sed -i -e '/JEMALLOC_HAVE_SECURE_GETENV/d' third_party/jemalloc.BUILD \
```

Comment:
can you comment on the changes you had to make under L68? so that we know if we need to keep these tweaks long term etc.